src/app/loginComponents/accounts/external/accounts.component.scss

```diff
@@ -2,44 +2,7 @@
 @import '/src/materialColorScheme.scss';
 @import '/src/styles.scss';
 
-// Resizing the logos to fit nicely in the card
-.account-logo {
-  width: 3.5em;
-  margin-right: 1em;
-}
-
-// Style for the token form field
-.token-form-field {
-  width: 300px;
-  font-size: 12px;
-}
-
 // Positioning the badge to the upper right corner of each logo
 .mat-badge-medium.mat-badge-above .mat-badge-content {
-  top: 0px;
-  left: 40px;
-  background: $green-color;
```

Comment:
actually the wrong color, this is the green-color used by workflows, whereas the checkmark and verified one is the same color as the one Charles introduced in the stepper (accent-3-dark)